src/TRoom.cpp

```diff
@@ -115,72 +114,66 @@ bool TRoom::hasExitStub(int direction)
 
 void TRoom::setExitStub(int direction, bool status)
 {
-    if(status)
-    {
-        if( ! hasExit(direction) )
-        {
-            if( ! exitStubs.contains(direction) )
-            {
+    if (status) {
+        if (!hasExit(direction)) {
+            if (!exitStubs.contains(direction)) {
                 // Previous code did not check for an existing entry for given
                 // direction and QList container permits duplicates of same value!
                 exitStubs.append(direction);
             }
-        }
-        else
-        {
-            QString error = QString("Set exit stub in given direction in RoomID(%1) - there is already an exit there!").arg( id );
+        } else {
+            QString error = QString("Set exit stub in given direction in RoomID(%1) - there is already an exit there!").arg(id);
             mpRoomDB->mpMap->logError(error);
         }
-    }
-    else
+    } else {
         exitStubs.removeAll(direction);
+    }
 }
 
-int TRoom::getExitWeight(const QString& cmd )
+int TRoom::getExitWeight(const QString& cmd)
 {
-    if( exitWeights.contains( cmd ) )
-    {
+    if (exitWeights.contains(cmd)) {
         return exitWeights[cmd];
-    }
-    else
+    } else {
         return weight; // NOTE: if no exit weight has been set: exit weight = room weight
```

Comment:
:thinking:  Blast, I thought I had properly coded things so that the weight of the _destination_ room was used if there was not an exit weight.  Or may-haps it is already done by the code that uses the details so things are OK after all...!

Anyhow nothing to do about it here and now.